### PR TITLE
Rebuild packages for missing metadata vol. 5

### PR DESCRIPTION
**Pull request by Chocimier**
All packages was build for x86_64, x86_64-musl, armv7hf and aarch64-musl and passed check stage on x86_64 and x86_64-musl. Nothing was tested manually whether it still works.

libfirm also built for armv6hf-musl, as it's broken on aarch64-musl.

317 more packages remain to rebuild.

### CODE REVIEW

File: srcpkgs/libasr/template [hunk 1]
```diff
@@ -1,7 +1,8 @@
 # Template file for 'libasr'
 pkgname=libasr
 version=1.0.2
-revision=2
+revision=3
+wrksrc="${pkgname}-${pkgname}-${version}"
 build_style=gnu-configure
 hostmakedepends="automake libtool"
 makedepends="libressl-devel"
```
Comment (maxice8) [inline srcpkgs/libasr/template]: BSD-2-Clause

File: srcpkgs/libatasmart/template [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
+revision=5
 build_style=gnu-configure
 configure_args="--sbindir=/usr/bin --disable-static"
 hostmakedepends="automake libtool pkg-config"
 makedepends="eudev-libudev-devel"
 short_desc="ATA S.M.A.R.T. Reading and Parsing Library"
-homepage="http://0pointer.de/blog/projects/being-smart.html"
-license="LGPL-2.1"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
+license="LGPL-2.1"
```
Comment (maxice8) [inline srcpkgs/libatasmart/template]: LGPL-2.1-or-later

File: srcpkgs/libbs2b/template [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'libbs2b'
 pkgname=libbs2b
 version=3.1.0
-revision=2
+revision=3
 build_style=gnu-configure
 hostmakedepends="pkg-config"
 makedepends="libsndfile-devel"
```
Comment (maxice8) [inline srcpkgs/libbs2b/template]: seems to be purely MIT

File: srcpkgs/libcryptui/template [hunk 4]
```diff
@@ -1,18 +1,18 @@
...
-hostmakedepends="pkg-config intltool"
+hostmakedepends="pkg-config intltool dbus-glib-devel glib gnupg"
 makedepends="libSM-devel dbus-glib-devel gtk+3-devel gpgme-devel
- libnotify-devel libgnome-keyring-devel gnupg"
+ libnotify-devel libgnome-keyring-devel"
 depends="hicolor-icon-theme libgnome-keyring>=3.10 gnupg"
 short_desc="GNOME Interface components for OpenPGP"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
-homepage="http://www.gnome.org"
 license="GPL-2, LGPL-2.1"
```
Comment (maxice8) [inline srcpkgs/libcryptui/template]: GPL-2.0-or-later, LGPL-2.0-or-later

File: srcpkgs/libdaemon/template [hunk 5]
```diff
@@ -1,14 +1,14 @@
...
 pkgname=libdaemon
 version=0.14
-revision=8
+revision=9
 build_style=gnu-configure
 configure_args="--disable-static --disable-lynx"
 short_desc="Lightweight C library that eases the writing of UNIX daemons"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
-homepage="http://0pointer.de/lennart/projects/$pkgname"
 license="LGPL-2.1"
```
Comment (maxice8) [inline srcpkgs/libdaemon/template]: LGPL-2.1-or-later

File: srcpkgs/libdbusmenu-qt/template [hunk 6]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'libdbusmenu-qt'
 pkgname=libdbusmenu-qt
 version=0.9.2
-revision=2
+revision=3
 build_style=cmake
 hostmakedepends="doxygen qt-host-tools qt-devel"
 makedepends="qt-devel"
```
Comment (maxice8) [inline srcpkgs/libdbusmenu-qt/template]: GPL-3.0-or-later

File: srcpkgs/libee/template [hunk 7]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'libee'
 pkgname=libee
 version=0.4.1
-revision=3
+revision=4
 build_style=gnu-configure
 configure_args="--sbindir=/usr/bin --disable-static"
```
Comment (maxice8) [inline srcpkgs/libee/template]: --sbindir= is redundant here

Comment (maxice8) [reply srcpkgs/libee/template]: LGPL-2.1-or-later

File: srcpkgs/libfirm/template [hunk 8]
```diff
@@ -1,17 +1,21 @@
...
 version=1.22.0
-revision=1
+revision=2
 wrksrc="${pkgname}-${pkgname}-${version}"
 build_style=gnu-makefile
 hostmakedepends="perl python"
 short_desc="Graph based SSA intermediate code representation"
-maintainer="Christian Neukirchen <chneukirchen@gmail.com>"
+maintainer="Leah Neukirchen <leah@vuxu.org>"
 license="LGPL-2.1"
```
Comment (maxice8) [inline srcpkgs/libfirm/template]: LGPL-2.1-or-later

File: srcpkgs/libgee/template [hunk 9]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'libgee'
 pkgname=libgee
 version=0.6.8
-revision=5
+revision=6
 build_style=gnu-configure
 configure_args="--disable-static"
 hostmakedepends="pkg-config intltool glib-devel"
```
Comment (maxice8) [inline srcpkgs/libgee/template]: LGPL-2.1-or-later

File: srcpkgs/libgnome-keyring/template [hunk 10]
```diff
@@ -1,15 +1,16 @@
...
+revision=2
 build_style=gnu-configure
 configure_args="$(vopt_enable gir introspection)"
 hostmakedepends="pkg-config intltool $(vopt_if gir gobject-introspection)"
 makedepends="dbus-devel libglib-devel libgcrypt-devel"
+checkdepends="glib-devel"
 short_desc="GNOME keyring client library"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
-homepage="http://www.gnome.org"
 license="GPL-2, LGPL-2.1"
```
Comment (maxice8) [inline srcpkgs/libgnome-keyring/template]: GPL-2.0-or-later, LGPL-2.0-or-later

File: srcpkgs/libgnomecanvas/template [hunk 11]
```diff
@@ -1,15 +1,15 @@
...
+revision=9
 build_style=gnu-configure
 configure_args="--disable-static"
 hostmakedepends="pkg-config intltool glib-devel"
 makedepends="gtk+-devel libart-devel"
 short_desc="GNOME Canvas library"
-homepage="http://gnome.org/"
-license="LGPL-2"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
+license="LGPL-2"
```
Comment (maxice8) [inline srcpkgs/libgnomecanvas/template]: LGPL-2.0-or-later

File: srcpkgs/libguess/template [hunk 12]
```diff
@@ -1,14 +1,14 @@
...
 version=1.2
-revision=1
+revision=2
 build_style=gnu-configure
 hostmakedepends="pkg-config"
 makedepends="libmowgli-devel"
 short_desc="High-speed character set detection library"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
-homepage="http://www.atheme.org/project/libguess"
 license="3-clause-BSD"
```
Comment (maxice8) [inline srcpkgs/libguess/template]: BSD-3-Clause

File: srcpkgs/libieee1284/template [hunk 13]
```diff
@@ -1,14 +1,14 @@
...
 version=0.2.11
-revision=3
+revision=4
 only_for_archs="i686 i686-musl x86_64 x86_64-musl"
 build_style=gnu-configure
 configure_args="--disable-static --without-python"
 short_desc="A library to query devices connected in parallel port"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
-homepage="http://cyberelk.net/tim/libieee1284"
 license="GPL-2"
```
Comment (maxice8) [inline srcpkgs/libieee1284/template]: GPL-2.0-or-later

File: srcpkgs/libmad/template [hunk 14]
```diff
@@ -1,13 +1,13 @@
...
 pkgname=libmad
 version=0.15.1b
-revision=8
+revision=9
 build_style=gnu-configure
 hostmakedepends="automake pkg-config"
 short_desc="High-quality MPEG audio decoder"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
-homepage="http://www.underbit.com/products/mad/"
 license="GPL-2"
```
Comment (maxice8) [inline srcpkgs/libmad/template]: GPL-2.0-or-later

File: srcpkgs/libmcrypt/template [hunk 15]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'libmcrypt'
 pkgname=libmcrypt
 version=2.5.8
-revision=2
+revision=3
 build_style=gnu-configure
 short_desc="A data encryption library"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
```
Comment (maxice8) [inline srcpkgs/libmcrypt/template]: GPL-2.0-or-later

File: srcpkgs/libmms/template [hunk 16]
```diff
@@ -1,14 +1,14 @@
...
-revision=1
+revision=2
 build_style=gnu-configure
 hostmakedepends="pkg-config"
 makedepends="libglib-devel"
 short_desc="MMS stream protocol library"
-homepage="http://sourceforge.net/projects/libmms/"
-license="LGPL-2.1"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
+license="LGPL-2.1"
```
Comment (maxice8) [inline srcpkgs/libmms/template]: LGPL-2.0-or-later

File: srcpkgs/libmpd/template [hunk 17]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'libmpd'
 pkgname=libmpd
 version=11.8.17
-revision=2
+revision=3
 build_style=gnu-configure
 hostmakedepends="pkg-config"
 makedepends="libglib-devel"
```
Comment (maxice8) [inline srcpkgs/libmpd/template]: GPL-2.0-or-later

File: srcpkgs/libnfnetlink/template [hunk 18]
```diff
@@ -1,12 +1,12 @@
 # Template file for 'libnfnetlink'
 pkgname=libnfnetlink
 version=1.0.1
-revision=3
+revision=4
 build_style=gnu-configure
+short_desc="Low-level library for netfilter kernel/userspace communication"
 maintainer="Orphaned <orphan@voidlinux.eu>"
 license="GPL-2"
```
Comment (maxice8) [inline srcpkgs/libnfnetlink/template]: GPL-2.0-only

File: srcpkgs/libpano13/template [hunk 19]
```diff
@@ -1,17 +1,18 @@
...
-revision=1
+revision=2
+wrksrc="libpano13-${version}"
+build_style=gnu-configure
+hostmakedepends="pkg-config"
+makedepends="zlib-devel libpng-devel libjpeg-turbo-devel tiff-devel"
+checkdepends="perl"
 short_desc="Panorama Pictures Tools library"
 maintainer="Martin Riese <grauehaare@gmx.de>"
 license="GPL-2"
```
Comment (maxice8) [inline srcpkgs/libpano13/template]: GPL-2.0-or-later

File: srcpkgs/libquvi/template [hunk 20]
```diff
@@ -1,18 +1,18 @@
...
-revision=7
+revision=8
 build_style=gnu-configure
 hostmakedepends="pkg-config"
 makedepends="lua52-devel libcurl-devel libquvi-scripts"
 depends="libquvi-scripts>=${version}"
-replaces="quvi<0.4.0"
 short_desc="C library that can be used to parse flash media stream URLs"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
 license="LGPL-2.1"
```
Comment (maxice8) [inline srcpkgs/libquvi/template]: LGPL-2.1-or-later

File: srcpkgs/librlog/template [hunk 21]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'librlog'
 pkgname=librlog
 version=1.4
-revision=4
+revision=5
 wrksrc="rlog-$version"
 build_style=gnu-configure
 short_desc="Flexible message logging facility for C++ programs and libraries"
```
Comment (maxice8) [inline srcpkgs/librlog/template]: LGPL-2.1-or-later

File: srcpkgs/libshout/template [hunk 22]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'libshout'
 pkgname=libshout
 version=2.4.1
-revision=1
+revision=2
 build_style=gnu-configure
 makedepends="libtheora-devel libvorbis-devel speex-devel"
 short_desc="Icecast client library"
```
Comment (maxice8) [inline srcpkgs/libshout/template]: LGPL-2.0-or-later

File: srcpkgs/libspiro/template [hunk 23]
```diff
@@ -1,11 +1,11 @@
...
 pkgname=libspiro
 version=0.5.20150702
-revision=1
+revision=2
 build_style=gnu-configure
 hostmakedepends="automake libtool"
 short_desc="Simplifies the drawing of beautiful curves"
-maintainer="Christian Neukirchen <chneukirchen@gmail.com>"
+maintainer="Leah Neukirchen <leah@vuxu.org>"
 license="GPL-3"
```
Comment (maxice8) [inline srcpkgs/libspiro/template]: GPL-3.0-or-later

File: srcpkgs/libstatgrab/template [hunk 24]
```diff
@@ -1,15 +1,15 @@
...
-revision=2
+revision=3
 build_style=gnu-configure
 configure_args="--disable-static"
 hostmakedepends="autoconf"
 makedepends="ncurses-devel"
 short_desc="Library being useful interface to system statistics"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
-homepage="http://www.i-scream.org/libstatgrab"
 license="LGPL-2.1, GPL-2"
```
Comment (maxice8) [inline srcpkgs/libstatgrab/template]: LGPL-2.1-or-later, GPL-2.0-or-later

File: srcpkgs/libunique/template [hunk 25]
```diff
@@ -1,15 +1,15 @@
...
-revision=6
+revision=7
 build_style=gnu-configure
 configure_args="--enable-bacon=yes --enable-dbus=yes"
 hostmakedepends="pkg-config glib-devel"
 makedepends="dbus-glib-devel gtk+3-devel"
 short_desc="Library for writing single instance applications"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
-homepage="http://live.gnome.org/LibUnique"
 license="LGPL-2.1"
```
Comment (maxice8) [inline srcpkgs/libunique/template]: LGPL-2.1-or-later

File: srcpkgs/libunique1/template [hunk 26]
```diff
@@ -1,19 +1,19 @@
...
 wrksrc="libunique-${version}"
-patch_args="-Np1"
 build_style=gnu-configure
 configure_args="--disable-static --disable-dbus $(vopt_if gir gobject-introspection)"
 hostmakedepends="libtool automake pkg-config glib-devel"
 makedepends="libSM-devel gtk+-devel"
 short_desc="Library for writing single instance applications (GTK+2)"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
-homepage="http://live.gnome.org/LibUnique"
 license="LGPL-2.1"
```
Comment (maxice8) [inline srcpkgs/libunique1/template]: LGPL-2.1-or-later

File: srcpkgs/libusb-compat/template [hunk 27]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'libusb-compat'
 pkgname=libusb-compat
 version=0.1.5
-revision=6
+revision=7
 build_style=gnu-configure
 hostmakedepends="pkg-config"
 makedepends="libusb-devel"
```
Comment (maxice8) [inline srcpkgs/libusb-compat/template]: LGPL-2.1-or-later

File: srcpkgs/libutempter/template [hunk 28]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'libutempter'
 pkgname=libutempter
 version=1.1.6
-revision=4
+revision=5
 build_style=gnu-makefile
 short_desc="Library interface to record user sessions in utmp/wtmp files"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
```
Comment (maxice8) [inline srcpkgs/libutempter/template]: LGPL-2.1-or-later

File: srcpkgs/libvisual/template [hunk 29]
```diff
@@ -1,13 +1,13 @@
...
 pkgname=libvisual
 version=0.4.0
-revision=8
+revision=9
 build_style=gnu-configure
 hostmakedepends="pkg-config"
 short_desc="Abstraction library for audio visualisation plugins"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
-homepage="http://sourceforge.net/projects/libvisual/"
 license="LGPL-2.1"
```
Comment (maxice8) [inline srcpkgs/libvisual/template]: LGPL-2.1-or-later

File: srcpkgs/libwnck2/template [hunk 30]
```diff
@@ -1,16 +1,16 @@
...
+revision=6
 wrksrc="${pkgname/2/}-${version}"
 build_style=gnu-configure
 configure_args="$(vopt_enable gir introspection)"
 hostmakedepends="pkg-config intltool gdk-pixbuf-devel $(vopt_if gir gobject-introspection)"
 makedepends="startup-notification-devel libXres-devel gtk+-devel"
 short_desc="Library for layout and rendering of text (GTK+2)"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
-homepage="http://www.gnome.org/"
 license="LGPL-2.1"
```
Comment (maxice8) [inline srcpkgs/libwnck2/template]: LGPL-2.1-or-later

File: srcpkgs/libxfce4ui/template [hunk 31]
```diff
@@ -1,13 +1,13 @@
...
 build_style=gnu-configure
 configure_args="--disable-static"
+conf_files="/etc/xdg/xfce4/xfconf/xfce-perchannel-xml/xfce4-keyboard-shortcuts.xml"
 hostmakedepends="xfce4-dev-tools pkg-config intltool glib-devel gettext-devel"
 makedepends="gtk+-devel gtk+3-devel libxfce4util-devel xfconf-devel dbus-glib-devel
  libxml2-devel startup-notification-devel libSM-devel"
-conf_files="/etc/xdg/xfce4/xfconf/xfce-perchannel-xml/xfce4-keyboard-shortcuts.xml"
 short_desc="Replacement of the old libxfcegui4 library"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
 license="GPL-2, LGPL-2.1"
```
Comment (maxice8) [inline srcpkgs/libxfce4ui/template]: LGPL-2.0-or-later

File: srcpkgs/libxfce4util/template [hunk 32]
```diff
@@ -1,8 +1,9 @@
 # Template file for 'libxfce4util'
 pkgname=libxfce4util
 version=4.12.1
-revision=2
+revision=3
 build_style=gnu-configure
+configure_args="--sbindir=/usr/bin"
```
Comment (maxice8) [inline srcpkgs/libxfce4util/template]: redundant

File: srcpkgs/libxfce4util/template [hunk 33]
```diff
@@ -1,8 +1,9 @@
...
 hostmakedepends="pkg-config intltool"
 makedepends="libglib-devel"
 short_desc="Utility library for the Xfce4 desktop environment"
```
Comment (maxice8) [inline srcpkgs/libxfce4util/template]: LGPL-2.0-or-later

File: srcpkgs/libzapojit/template [hunk 34]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'libzapojit'
 pkgname=libzapojit
 version=0.0.3
-revision=5
+revision=6
 build_style=gnu-configure
 configure_args="--disable-static"
 hostmakedepends="pkg-config intltool $(vopt_if gir gobject-introspection)"
```
Comment (maxice8) [inline srcpkgs/libzapojit/template]: LGPL-2.1-or-later

**Comment by maxice8** [comment]
Review finished

**Comment by Chocimier** [comment]
Updated

**Comment by maxice8** [comment]
Merged

**Pull request closed without merging**